pkg/server/event_handler_test.go

```diff
@@ -325,18 +404,34 @@ func newTestServer(handler func(w http.ResponseWriter, r *http.Request)) (*http.
 	}
 }
 
+// Returns fake metadata that include github resource info.
+func getFakeMetadata() []byte {
```

Comment:
https://github.com/abcxyz/readability/wiki/Go-Best-Practices#test-helper
```suggestion
func testGCSMetadataBytes() []byte {
```